pkg/covermerger/covermerger_test.go

```diff
@@ -313,6 +317,7 @@ samp_time,1,360,arch,b1,ci-mock,git://repo,master,commit2,not_changed.c,func1,4,
 			[
 				{
 					"FilePath":"not_changed.c",
+					"FuncName":"func1",
```

Comment:
Can we have different FuncNames somewhere to ensure checkedFuncName() is working?

Reply:
Let's test the function itself.